BEIMA.Client/src/pages/Devices/DevicePage.js

```diff
@@ -38,14 +43,17 @@ const DevicePage = () => {
       if(device.location.buildingId !== null){
         building = (await GetBuilding(device.location.buildingId)).response;
       } else {
-        building = {name: 'No Assigned Building'};
+        building = noBuildingObj;
```

Comment:
We should keep this and make a new variable if need, this is for the device title and if there is no building, we should have it say No Assigned Building rather than Select Building

Reply:
fixed e679456